CHANGELOG.md

```diff
@@ -4,6 +4,43 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.17.1](https://github.com/go-gitea/gitea/releases/tag/1.17.1) - 2022-08-17
+
+* SECURITY
+  * Correctly escape within tribute.js (#20831) (#20832)
+* FEATURES
+  * Add support for NuGet API keys (#20721) (#20734)
```

Comment:
Interesting to see a feature in a patch version...

Reply:
I changed it to enhancement since it's based on NuGet implementation.